scripts/db_migrator.py

```diff
@@ -199,17 +210,16 @@ def version_1_0_3(self):
         if self.asic_type == "mellanox":
             if self.mellanox_buffer_migrator.mlnx_migrate_buffer_pool_size('version_1_0_3', 'version_1_0_4') and self.mellanox_buffer_migrator.mlnx_migrate_buffer_profile('version_1_0_3', 'version_1_0_4'):
                 self.set_version('version_1_0_4')
-        else:
-            self.set_version('version_1_0_4')
 
+        self.migrate_copp_table()
```

Comment:
Lets have copp table migration as part of` 1_0_4`? and set new as `1_0_5`

Reply:
202006 version had the release version as 1_0_3. Is it ok to bump up the version twice to 1_0_5 between releases?

Reply:
We can have this part of the current version so as to execute this always